docs/reference/search/search-your-data/retrieve-selected-fields.asciidoc

```diff
@@ -107,19 +79,20 @@ POST my-index-000001/_search
     Other field types do not support the `format` parameter.
 // end::fields-param-callouts[]
 
-The values are returned as a flat list in the `fields` section in each hit:
+The response includes values as a flat list in the `fields` section for each
+hit. Because the `fields` parameter doesn't fetch entire objects, only leaf
+fields are returned.
+
+NOTE: The `fields` response always returns an array of values for each field,
```

Comment:
In an earlier conversation with @nik9000, we thought it could be helpful to give this note a title. Otherwise it is easy to miss these important points.

Reply:
++

I'll highlight this better instead of just a note.

Reply:
Sure!

I've wanted to link folks to it and its nice if it has a title because then it'll have an anchor I can link to.